.github/workflows/runtimes-matrix.json

```diff
@@ -4,6 +4,7 @@
     "package": "kitchensink-runtime",
     "path": "substrate/frame",
     "header": "substrate/HEADER-APACHE2",
+    "bench_flags": "--genesis-builder-policy=none --exclude-pallets=pallet_xcm,pallet_xcm_benchmarks::fungible,pallet_xcm_benchmarks::generic,pallet_nomination_pools,pallet_remark,pallet_transaction_storage",
```

Comment:
@pgherveou the contracts pallet seems to require some genesis setup for the MBM testing.  
If i run it without genesis setup, then it errors here: 
https://github.com/paritytech/polkadot-sdk/blob/7ecf3f757a5d6f622309cea7f788e8a547a5dce8/substrate/frame/contracts/src/migration.rs#L90

Does the contracts-rococo runtime need to have some MBMs configured to make the benchmarks work? Not sure how it worked in the past, but am trying to get it working without genesis setup.

Reply:
Fixed